src/js/api/order.js

```diff
@@ -813,6 +813,14 @@ ripe.Ripe.prototype.readyOrderP = function(number, options) {
     });
 };
 
+/**

```

Comment:
Maybe since we're here we could update all the missing descriptions too

Reply:
I'll make another PR will all of this and will remove this change in this file